contracts/src/protocol/OmniPortalStorage.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity =0.8.24;
+
+import { XTypes } from "../libraries/XTypes.sol";
+import { IOmniPortal } from "../interfaces/IOmniPortal.sol";
+import { IOmniPortalAdmin } from "../interfaces/IOmniPortalAdmin.sol";
+
+/**
+ * @title OmniPortalStorage
+ * @notice Storage layout for OmniPortal
+ */
+abstract contract OmniPortalStorage is IOmniPortal, IOmniPortalAdmin {
+    /**
+     * @notice Chain ID of the chain to which this portal is deployed
+     */
+    uint64 public immutable chainId;
```

Comment:
since this isn't storage, should we keep it in the `OmniPortal`? Matching how we do it with `OmniAVS`

Reply:
Oh yes good point  (side note, I am thinking of removing this)